modules/dynamic-plug-ins-get-started.adoc

```diff
@@ -14,6 +14,14 @@ To get started using the dynamic plug-in, you must set up your environment to wr
 
 .Procedure
 
+. Visit the link:https://github.com/openshift/console-plugin-template[`console-plugin-template`] repository containing a template for creating plug-ins.
+
+. Select *Use this Template* to create a GitHub repository.
+
+. Re-name the template with the name of your plug-in.
+
+. From your copied repository, clone it to your local machine so you can edit the code.
+
```

Comment:
This is just adding back the [procedure from 4.10 ](https://docs.openshift.com/container-platform/4.10/web_console/dynamic-plug-ins.html#getting-started-with-dynamic-plugins_dynamic-plugins) to add the correct link to the template repo. I am concerned if we merge the Helm chart to 4.11 with new prerequisite step to ensure you have cloned the template without updating this too, users will be confused.

Reply:
Yes, this step is mandatory for the following steps for work (both for local development and using the Helm chart).